packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -139,7 +139,8 @@ Posts.addView("old", terms => ({
 
 Posts.addView("daily", terms => ({
   selector: {
-    baseScore: {$gt: terms.karmaThreshold || -100}
+    baseScore: {$gt: terms.karmaThreshold || -100},
+    unreviewedUser: {$in: [false,null]},
```

Comment:
When you add a field to a view, you need to also add it to the index for that view. Otherwise you'll get a pretty major performance regression.